src/screens/HomePage/HomePage.tsx

```diff
@@ -131,13 +147,19 @@ export default function HomePage() {
                 setMenuOpen={setMenuOpen}
               />
               <HomepageItems isLoading={isLoading} userRegions={userRegions} />
-              <Toggle
-                showCounties={showCounties}
-                onClickSwitch={onClickSwitch}
-              />
+              <ToggleWrapper>
```

Comment:
lets pull this wrapper + everything inside into its own separate component to keep HomePage.tsx concise